src/Sarif/Baseline/ResultMatching/DataStructures/ExtractedResult.cs

```diff
@@ -1,14 +1,17 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-
 namespace Microsoft.CodeAnalysis.Sarif.Baseline.ResultMatching
 {
     /// <summary>
-    /// Result metadata for matching a group of results from a 
-    /// set of runs (such as the current set of results) to a group of results 
-    /// from a different set of runs (such as the prior set of results).
+    /// This class associates a Result object with additional metadata that
+    /// makes it possible for a matching algorithm to determine if it matches
+    /// a result from another run.
```

Comment:
>run [](start = 30, length = 3)

As I mentioned in a comment on `SarifLogResultMatcher`, this class could also be called `MatchableResult`, and I think this comment conveys that better.